ocaml/xapi/create_misc.ml

```diff
@@ -387,7 +387,38 @@ let make_packs_info () =
 		in
 		Array.fold_left (fun l fname -> get_pack_details fname @ l) [] packs
 	with _ -> []
-  
+
+let trim s = String.filter_chars s (not ++ String.isspace)
+
+let make_xen_livepatch_list () = 
+	let lines = String.split '\n' (Helpers.get_process_output "/usr/sbin/xen-livepatch list") in
```

Comment:
Let's put the path of the script in xapi_globs.ml along with the other strings, and make it an option in the config file (same below).